composer.json

```diff
@@ -22,17 +22,37 @@
         "php": ">=7.2"
     },
     "require-dev": {
-        "ockcyp/covers-validator": "1.3.3",
-        "phpunit/phpunit": "^8.5.13||^9.5.0",
+        "ockcyp/covers-validator": "^1.3.3",
+        "phpunit/phpunit": "^6.5.14||^7.0.15||^8.5.13||^9.5.0",
```

Comment:
Just noting the largest change for 7.2 compatibility in CI is here. Our tests need to support 6.5+ which shouldn't be an issue